src/store/modules/farmer/selectors.ts

```diff
@@ -0,0 +1,26 @@
+import { createSelector } from 'reselect';
+import { Farmer } from './types';
+import { State } from '../../types';
+
+const getFarmerEntries = (state: State) => state.farmer.farmers;
+// const getCurrentFarmerUUID = (state: State) => state.currentFarmer.currentFarmerUUID; //need to merge with dairy/master for this
+
+/************Selectors for a specific farmer ***************/
+
+/**Selector for all farmers in the store */
+export const getAllFarmers = createSelector(
+  [getFarmerEntries],
+  (farmerEntries: Farmer[]) => farmerEntries);
```

Comment:
I am slightly confused regarding this, isn't this exactly the same as `getFarmerEntries`? Also changing to `getFarmerRows` for that function might be more in sync with how you access them, because it is now `state.farmer.rows`